### ISSUE TITLE
Add LED Circuits

### ISSUE THREAD

**Issue by o5akafeeva** [o5akafeeva/SAP-COMPUTER#2]
An array of transistor/LED circuits will be fed a voltage from the different Buses of the device to serve as a visual indication of the data at that point in the circuit. Next to these LED circuits I would also like a pin header footprint to connect 1X4 header strip for easy testing and interfacing with the bus. The suggested circuit shown here: 
https://noisehackerspace.com/wp-content/uploads/2017/10/transistor-npn-led-switch-circuit-animated.gif 
will be modified for an LED with a 2.1V drop and +5V power supply for a 16mA current the current limiting resistor can be (5-2.1)/0.0161 = 180 Ohms. The base of the NPN transistor will go to GND through a 1M resistor. The signal will go into the base as well through a (5-0.6)/(0.0161/hFE) =27K, if hFE = 100. The circuit needs to be tested on the breadboard and results posted here.

**Comment by o5akafeeva** [comment]
Another test to run, use those transistor/LED circuits with a breadboarded MAR chip and test the MAR with the LEDs, but without a transistor to see if it's even worth adding to the board.

**Comment by o5akafeeva** [comment]
Skipping the MAR chip test to save time, I breadboarded the transistor/LED circuit and found no noticeable difference in operation after removing the 1M resistor. Also the hFE of the transistor turned out to be 333 and the base current was being amplified to 53mA with 27K at the base. The current limiting resistor was in place and nothing blew up. I took the current limiting resistor off and used 180K for the base resistor. The transistor BE voltage was 0.73, making (5-0.73)/180K= 24uA current at the base, which when amplified gave 7-8 mA across the LED. This means that the transistor/LED circuit requires only 3 components 180K, Red LED and a 2N3904. For the schematic simply remove all the unnecessary components and leave those 3.
